coral-schema/src/main/java/com/linkedin/coral/schema/avro/RelToAvroSchemaConverter.java

```diff
@@ -374,7 +381,7 @@ private Schema getTableScanSchema(TableScan tableScan) {
         throw new RuntimeException("Cannot find table " + dbName + "." + tableName + " in Hive metastore");
       }
 
-      Schema tableSchema = SchemaUtilities.getAvroSchemaForTable(baseTable, strictMode, forceLowercase);
+      Schema tableSchema = SchemaUtilities.getAvroSchemaForTable(baseTable, strictMode, false);
```

Comment:
I think we can remove `forceLowercase` parameter directly from `SchemaUtilities.getAvroSchemaForTable` given we don't convert the schema to lowercase there anymore with this PR.

Reply:
This make sense, thanks for finding this. I will try to remove this parameter for this function